packages/block-editor/src/components/block-list/insertion-point.js

```diff
@@ -23,6 +23,24 @@ import { isRTL } from '@wordpress/i18n';
 import Inserter from '../inserter';
 import { getBlockDOMNode } from '../../utils/dom';
 
+function offsetIframe( rect, ownerDocument ) {
```

Comment:
This functions comes from https://github.com/WordPress/gutenberg/blob/e54a9ec2f97e28db3293628e921e40e81cfe5f76/packages/components/src/popover/index.js#L45

Is there a good package to host this function so we can use it in both `components` and `block-editor` package?

Reply:
I'm not sure if there's an clear candidate but maybe the `@wordpress/dom` package could be a fit.

Reply:
I took a brief look at Gutenberg packages and agree that `@wordpress/dom` might be the right place